src/opnsense/www/js/widgets/IpsecLeases.js

```diff
@@ -111,7 +111,7 @@ export default class IpsecLeases extends BaseTableWidget {
         // Prepare a summary row for user counts
         let userCountsRow = `
             <div>
-                <span><b>${this.translations.users}:</b> ${totalUsersCount} - <b>${this.translations.online}:</b> ${onlineUsersCount} - <b>${this.translations.offline}:</b> ${offlineUsersCount}</span>
+                <span>${this.translations.users}: ${totalUsersCount} | ${this.translations.online}: ${onlineUsersCount} | ${this.translations.offline}: ${offlineUsersCount}</span>
```

Comment:
With all of HTML and CSS at our disposal we still use "-" and "|" to delimit row data? :)

Reply:
I mean we could use some font awesome icon there as well but which one would look nice?

Reply:
Here are some suggestions:

https://fontawesome.com/icons/ellipsis-vertical?s=solid
https://fontawesome.com/icons/grip-lines-vertical?f=classic&s=solid
https://fontawesome.com/icons/minus?f=classic&s=solid
https://fontawesome.com/icons/grip-vertical?f=classic&s=solid

Any of them in your favor or do you have a different suggestion?

Reply:
column-based data should be divs or tables but I don't want to derail this PR

Reply:
Lets fix this later then. It has to be thought of in multiple widgets.